src/scripts/main.js

```diff
@@ -1,5 +1,36 @@
 'use strict';
 
-document.addEventListener('click', e => {
-  // write code here
+document.addEventListener('click', (event) => {
```

Comment:
Just rename `event` to get rid of this alert
<img width="985" alt="image" src="https://user-images.githubusercontent.com/99403198/210077160-e2cea9fa-399b-44d8-9913-deb3dec2fe01.png">

```suggestion
document.addEventListener('click', (e) => {
```